### ISSUE TITLE
Less blurry blur effect

### ISSUE THREAD

**Issue by leonlywong** [romannurik/muzei#531]
Even applying the minimal amount of blur effect is too blurry for me, glad if you could reduce the minimum amount of blur I could add. Thanks.

**Comment by jaminroe** [comment]
I agree on this. Would be nice if the effect from the bottom of the slider would be more subtle
